src/components/home/common/contactForm/modal.css

```
@@ -0,0 +1,94 @@
+
+.contact-modal {
+  top: 0;
+  left: 0;
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  background-color: var(--background-open-modal);
+}
+
+.contact-form-headline {
+  text-align: center;
+}
+
+.empty-fields-error {
+  border: 1px solid  var( --red-color);
+}
+
+.empty-fields-error-message {
+  color: var(--red-color);
+  margin: 0em 0 0.5em 0;
+  letter-spacing: 0.1em;
+  font-size: 0.8em;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  position: relative;
+  top: 10%;
+  width: 51em;
+  height: 35em;
+
+  @media only screen and (max-width: 767px){
+    width: 90%;
+  }
+}
+
+.contact-form--input-column {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.contact-form--input-row {
+  width: 50%;
+
+  @media only screen and (max-width: 767px){
+    width: 90%;
+  }
+}
+
+.contact-form--section {
+  margin: 2em 0 2em 0;
```

When top and bottom margins (also works for padding) are the same, you can use the 2 parameter shortcut in which the first indicates top/bottom and the second indicates left/right margins. See more info [here](https://developer.mozilla.org/es/docs/Web/CSS/margin)
```suggestion
  margin: 2em 0;
```